test/multiple-entrypoints.test.js

```diff
@@ -0,0 +1,68 @@
+const { loadPolicy } = require('../src/opa.js');
+const { readFileSync } = require('fs');
+
+const scenarios = [
+    { note: 'should run with default entrypoint', entrypoint: null },
+    { note: 'should run with numbered entrypoint specified', entrypoint: 0 },
+    { note: 'should run with named entrypoint specified', entrypoint: 'string/entrypoint' },
+    { note: 'should not run with entrypoint as object', entrypoint: {}, throws: 'entrypoint value is an invalid type, must be either string or number!' },
+    { note: 'should not run if entrypoint string does not exist', entrypoint: 'not/a/real/entrypoint', throws: 'entrypoint not/a/real/entrypoint is not valid in this instance!' },
+];
+
+describe('multiple entrypoints', () => {
+    let policy = null; 
+
+    beforeAll(async () => {
+        policy = await loadPolicy(readFileSync('test/fixtures/multiple-entrypoints.wasm.test'));

```

Comment:
💭 I suppose alternatively, we could compile this on the fly in the test. That would have the advantage that it we would always test against he wasm file produced by the OPA version used in the test.

However, if we ever want to support different ABI versions here, it would make sense to pin this one down now, and commit the file to git. Would you mind documenting the call used to create that file in some place then? A makefile, perhaps...? 🤔

Reply:
I added the binary generation to the beforeAll, it seemed like a long command to have on a pretest stage after I added the folder pathing. It will fail if the opa binary is not located on path. If we would rather it skip the tests if the binary is not found let me know and I can change that up.